<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location Data browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_main">
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_LOC">
        <attribute name="title">
         <string>List of countries</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>50</y>
           <width>311</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_LOC">
          <item>
           <widget class="QLabel" name="label_LOC_01">
            <property name="text">
             <string>Select continent :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_LOC"/>
          </item>
         </layout>
        </widget>
        <widget class="QScrollArea" name="scrollArea_LOC">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>751</width>
           <height>341</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>749</width>
            <height>339</height>
           </rect>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>9</y>
             <width>731</width>
             <height>321</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_LOC">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_LOC"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="search_button_LOC">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>70</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_SO">
        <attribute name="title">
         <string>Search Location</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>211</x>
           <y>91</y>
           <width>361</width>
           <height>111</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_SO_1">
          <item>
           <widget class="QLabel" name="label_SO">
            <property name="text">
             <string>Select location :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_SO_Continent"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_SO_Country"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>150</y>
           <width>61</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_SO_2">
          <item>
           <widget class="QLabel" name="label_SO_Continent">
            <property name="text">
             <string>Continent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_SO_Country">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="search_button_SO">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>230</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SO_research_return">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>290</y>
           <width>511</width>
           <height>131</height>
          </rect>
         </property>
         <property name="text">
          <string>Waiting for research return...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
